sphinx/themes/basic/static/searchtools.js_t

```diff
@@ -268,11 +276,7 @@ var Search = {
             displayNextItem();
           });
         } else if (DOCUMENTATION_OPTIONS.HAS_SOURCE) {
-          var suffix = DOCUMENTATION_OPTIONS.SOURCELINK_SUFFIX;
-          if (suffix === undefined) {
-            suffix = '.txt';
-          }
-          $.ajax({url: DOCUMENTATION_OPTIONS.URL_ROOT + '_sources/' + item[5] + (item[5].slice(-suffix.length) === suffix ? '' : suffix),
+          $.ajax({url: DOCUMENTATION_OPTIONS.URL_ROOT + item[0] + '.html',
```

Comment:
We have to give a name of extension as a template variable. Sometimes it might be changed by `html_file_suffix`.
http://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_file_suffix